Gemfile.lock

```diff
@@ -342,3 +340,6 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   watir-webdriver
   web-console (~> 2.0)
+
+BUNDLED WITH
```

Comment:
ooh actually can we remove this? it makes minor version conflicts in different people's bundlers a barrier to entry